index.js

```diff
@@ -57,7 +57,7 @@ function getScriptType (fname, packageType) {
 }
 
 async function findPlugins (dir, options, accumulator = [], prefix) {
-  const { indexPattern, ignorePattern, scriptPattern } = options
+  const { indexPattern, ignorePattern, scriptPattern, dirNameRoutePrefix = true } = options
```

Comment:
Shouldn't this default to `false`?

Reply:
the default behaviour is currently true - explicitly setting the option to false restore the prior behaviour of *not* doing this

Reply:
/me goes back to knowing nothing of this module.

Reply:
hahahah - the behaviour changed from v2 to v3 to automatically being named after the folder. It's a cool feature but it needs an opt-out imo, especially for migration purposes